File: ce/snippets/visualbasic/CRMV8/queryvb/vb/createalinqquery3.vb
' <snippetcreatealinqquery3>


 Dim accountsWithCounty As Integer = ( _
     From a In svcContext.AccountSet _
     Where (a.Address1_County IsNot Nothing) _
     Select New Account With
            {
                .Name = a.Name,
                .Address1_City = a.Address1_City
            }
        ).ToArray().Count()
 Console.Write("Number of accounts with a county specified: ")
 Console.WriteLine(accountsWithCounty)

' </snippetcreatealinqquery3>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/query/cs/createalinqquery3.cs
// <snippetcreatealinqquery3>


 int accountsWithCounty = (from a in svcContext.AccountSet
                           where (a.Address1_County != null)
                           select new Account
                           {
                            Name = a.Name,
                            Address1_City = a.Address1_City
                           }).ToArray().Count();
 System.Console.Write("Number of accounts with a county specified: ");
 System.Console.WriteLine(accountsWithCounty);

// </snippetcreatealinqquery3>